pkg/operator2/secret.go

```diff
@@ -27,6 +27,14 @@ func (c *authOperator) expectedSessionSecret() (*corev1.Secret, error) {
 	return secret, nil
 }
 
+func (c *authOperator) bootstrapUserExists() (bool, error) {
```

Comment:
The contents of this function now directly map to the conditions above, so everything is done twice - you can just remove this

Reply:
Yes. I removed this..